Week06/src/MainInventory.java

```diff
@@ -0,0 +1,152 @@
+        ProductList.add(new Vegetal("Fungus", 23, 5, 3, 100, "2200.01.01", "P,L,M"));
+
+        String option = "";
+        option = sc.next();
+        sc.nextLine();
+
+        while (!option.equals("4")){
+
+            switch (option){
+
+                case ("1"):
+                    for (Product a : ProductList){
+                        a.listProducts();
+                    }
+                    System.out.println("What kind of product do you want to add? 1 - animal or 2 - vegetal");
+                    String productType = sc.nextLine();
+                    System.out.println("Care este ID-ul produsului?");
+                    int id = sc.nextInt();
+                    System.out.println("Denumirea produsului?");
+                    String name = sc.next();
+                    System.out.println("Ce pret are?");
+                    int price = sc.nextInt();
+                    System.out.println("Ce greutate?");
+                    int weight = sc.nextInt();
+                    System.out.println("Care este cantitatea adaugata la stoc?");
+                    int quantity = sc.nextInt();
+                    System.out.println("Data expirarii? foloseste formatul YYYY.MM.DD");
+                    String expirationDate = sc.next();
+                    switch (productType){
+                        case ("1"):
+                            System.out.println("Temperatura ideala de stocare?");
+                            int storageTemp = sc.nextInt();
+                            Animal a = new Animal(name, id, price, weight, quantity, expirationDate, storageTemp);
+                            ProductList.add(a);
+                            a.listProducts();
+                            break;
+                        case ("2"):
+                            System.out.println("Ce vitamine contine? folosteste formatul X, Y, Z,...");
+                            String vitaminList = sc.nextLine();
+                            Vegetal b = new Vegetal(name, id, price, weight, quantity, expirationDate, vitaminList);
+                            ProductList.add(b);
+                            b.listProducts();
+                            break;
+                        default:
+                            System.out.println("N-ai introdus o valoare valida, mai incearca");
+                    }
+                    for (Product product : ProductList) {
+                        if (product.id == id) {
+                                product.quantity = product.quantity + quantity;
+                                // dayCounter++;
+                                break;
+
+                        }
+                    }
+
+                    Order addOrder = new Order(id, quantity);
+                    System.out.println("Data tranzactiei");
+                    addOrder.orderDate = sc.next();
+                    OrderList.add(addOrder);
```

Comment:
how can you tell if an order is to add product or to sell product? don't they look the same?